ocean_lib/ocean/__init__.py

```diff
@@ -1,4 +0,0 @@
-#
```

Comment:
There are unit tests as subdirectories of this. So I'm pretty sure that this one is needed in order for pytest. (But I could be wrong! Maybe check)